src/openrct2/ride/Vehicle.cpp

```diff
@@ -7448,12 +7444,7 @@ static bool loc_6DB38B(rct_vehicle * vehicle, rct_tile_element * tileElement)
     sint32 trackType   = track_element_get_type(tileElement);
     sint32 vangleStart = TrackDefinitions[trackType].vangle_start;
 
-    if (vangleStart != _vehicleVAngleEndF64E36 || bankStart != _vehicleBankEndF64E37)
-    {
-        return false;
-    }
-
-    return true;
+    return !(vangleStart != _vehicleVAngleEndF64E36 || bankStart != _vehicleBankEndF64E37);
```

Comment:
Why not making this better readable and simpeler? Or do I miss something here
`return vangleStart == _vehicleVAngleEndF64E36 && bankStart == _vehicleBankEndF64E37`

Reply:
Looks like you're right. ```!(!a || !b) === a && b```.

Reply:
For what it's worth, this is one of [De Morgan's laws](https://en.wikipedia.org/wiki/De_Morgan%27s_laws#Formal_notation).